pyzx/circuit/__init__.py

```diff
@@ -259,14 +259,24 @@ def from_graph(g:BaseGraph, split_phases:bool=True) -> 'Circuit':
         from .graphparser import graph_to_circuit
         return graph_to_circuit(g, split_phases=split_phases)
 
-    def to_graph(self, zh:bool=False, compress_rows:bool=True, backend:Optional[str]=None) -> BaseGraph:
+    def to_graph(
+        self,
+        zh:bool=False,
+        compress_rows:bool=True,
+        backend:Optional[str]=None,
+        init:Optional[List[bool]]=None,
+        post_select:Optional[List[int]]=None
+    ) -> BaseGraph:
         """Turns the circuit into a ZX-Graph.
         If ``compress_rows`` is set, it tries to put single qubit gates on different qubits,
-        on the same row."""
+        on the same row.
+        ``init`` denotes whether each input should be connected to |0\ranlge,
+        ``post_select`` denotes for each measurement whether it should be 
+        postselected to |0\rangle (0) or |1\ranlge (1)."""
```

Comment:
Just wanted to check: Is `|0>` or `|1>` correct, or would the measurement actually postselect to `|+>` or `|->` with a `VertexType.X` as you have it?